flink/v1.20/flink-runtime/LICENSE

```diff
@@ -464,47 +385,76 @@ License text:
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Project Nessie with the following in its NOTICE
-file:
+This binary artifact contains Project Nessie.
 
-| Dremio
-| Copyright 2015-2017 Dremio Corporation
-|
-| This product includes software developed at
-| The Apache Software Foundation (http://www.apache.org/).
+Copyright: Copyright 2015-2025 Dremio Corporation
+Home page: https://projectnessie.org/
+License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary includes code from Apache Commons.
-
-* Core ArrayUtil.
+This binary artifact contains failsafe.
 
-Copyright: 2020 The Apache Software Foundation
-Home page: https://commons.apache.org/
-License: https://www.apache.org/licenses/LICENSE-2.0
+Copyright: Jonathan Halterman and friends
+Home page: https://failsafe.dev/
+License: https://www.apache.org/licenses/LICENSE-2.0.html
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Apache HttpComponents Client.
+This binary artifact contains Codahale Metrics.
 
-Copyright: 1999-2022 The Apache Software Foundation.
-Home page: https://hc.apache.org/
-License: http://www.apache.org/licenses/LICENSE-2.0
+Copyright: Copyright (c) 2010-2013 Coda Hale, Yammer.com, 2014-2021 Dropwizard Team
+Home page: https://metrics.dropwizard.io/4.2.0/
+License: https://www.apache.org/licenses/LICENSE-2.0.html
 
 --------------------------------------------------------------------------------
 
-This product includes code from Apache HttpComponents Client.
-
-* retry and error handling logic in ExponentialHttpRequestRetryStrategy.java
```

Comment:
I'm not sure we should be removing this? We use this in the REST client which is part of core, HttpClient

Reply:
Yeah, I think this note is to state that source code was copied.